src/dbus_api/api.rs

```diff
@@ -82,18 +82,18 @@ fn object_path_to_pair(dbus_context: &DbusContext,
 }
 
 /// Get name for pool from object path
-fn object_path_to_pool_uuid(dbus_context: &DbusContext,
+fn pool_object_path_to_pair(dbus_context: &DbusContext,
                             path: &dbus::Path)
-                            -> Result<Uuid, (MessageItem, MessageItem)> {
-    let uuid = match dbus_context.pools.borrow().get(path) {
+                            -> Result<(dbus::Path<'static>, Uuid), (MessageItem, MessageItem)> {
+    let pair = match dbus_context.pools.borrow().get(path) {
```

Comment:
I really want to keep it. It is a very low-cost uniformity, quite meaningful, and will be very convenient when we drop these look-aside tables entirely (when the new dbus-rs release arrives). When this happens we will decorate the object path with exactly one type of data, what the dbus library allows. If that data is the same for all object paths it will reduce error-handling code by a lot.